modules/ROOT/pages/bare-metal.adoc

```diff
@@ -14,7 +14,7 @@ Before installing FCOS, you must have an Ignition configuration file and host it
 
 To install FCOS onto bare metal using the live ISO, follow these steps:
 
-- Download the ISO image:
+- Take the latest ISO image from the https://getfedora.org/coreos/download?tab=metal_virtualized&stream=stable[download page] or using podman (download stable by default, see https://coreos.github.io/coreos-installer/cmd/download/[documentation] for options):
```

Comment:
How about this?

```suggestion
- Download the latest ISO image from the https://getfedora.org/coreos/download?tab=metal_virtualized&stream=stable[download page] or with podman (see https://coreos.github.io/coreos-installer/cmd/download/[documentation] for options):
```

And then let's add `-s stable` to the `podman` options.  Hopefully that'll provide a hint on how to select alternate streams, without emphasizing it too much.

Reply:
better, thanks!

Reply:
Hmm, it looks like you opted not to add `-s stable`?

Reply:
Ah, I considered the update to the CMD documentation as separate. I am not sure how I do that.

Reply:
Oh, I just meant in the example immediately following this line.

Reply:
Ah, now I understand what you meant.

More relevant in this context would be the (also default) `-p metal`.
So I have added them both for consistency with the URL to the download page that I used.